packages/nuxt/src/app/composables/ssr.ts

```diff
@@ -17,6 +17,13 @@ export function useRequestHeaders (include?: any[]) {
   return Object.fromEntries(include.map(key => key.toLowerCase()).filter(key => headers[key]).map(key => [key, headers[key]]))
 }
 
+export function useRequestHeader(include: string) {
+  if (import.meta.client) { return '' }
+  const event = useRequestEvent()
+  const header = event ? getRequestHeader(event, include) : ''
+  return header
```

Comment:
```suggestion
  if (import.meta.client) { return undefined }
  const event = useRequestEvent()
  return event ? getRequestHeader(event, include) : undefined
```

Reply:
Should this return undefined or an empty string?

Reply:
I would prefer returning an empty string for consistency.

Reply:
I think undefined is mroe consistent as `getRequestHeader` returns undefined if a header doesn't exist, as does `useRequestHeaders(['unknown-header'])['unknown-header']`.